### PR TITLE
Recurrent archival of Cluster standup cryptpad

### PR DESCRIPTION
**Pull request by hsanjuan**

### CODE REVIEW

File: meeting-notes/2018/2018-05-24-to-2018-10-30--cluster-meeting-archive.md [hunk 1]
```diff
@@ -0,0 +1,1250 @@
...
+- clean up helm charts
+- document helm/kubernetes deployment 
+
+
+#### @mhz
+
+Last week: 
+
+- meetings with all the people
+  - tried to support David and Molly's IPFS roadmapping
```
Comment (diasdavid) [inline meeting-notes/2018/2018-05-24-to-2018-10-30--cluster-meeting-archive.md]: Thaaank you for that @meiqimichelle ❤️

File: meeting-notes/2018/2018-05-24-to-2018-10-30--cluster-meeting-archive.md [hunk 2]
```diff
@@ -0,0 +1,1250 @@
...
+
+- env var config discussion and PR
+- get back into metrics/tracing PR
+- extending benchmarks on go-libp2p-peerstore
+- started work on ipfs-cluster iptb docker plugin
+- reviewing tf code for eks infra
+
+This week:
+
+- start documenting metrics/tracing configuration
```
Comment (diasdavid) [inline meeting-notes/2018/2018-05-24-to-2018-10-30--cluster-meeting-archive.md]: Where can I find these?

File: meeting-notes/2018/2018-05-24-to-2018-10-30--cluster-meeting-archive.md [hunk 3]
```diff
@@ -0,0 +1,1250 @@
...
+### Updates since last time
+
+##### @hsanjuan
+
+Last week:
+
+- Headers for API
+- libp2p upgrade 
+- Tests with race (more failures and report)
+- Fetch gx from ipfs
```
Comment (diasdavid) [inline meeting-notes/2018/2018-05-24-to-2018-10-30--cluster-meeting-archive.md]: What does this mean?

File: meeting-notes/2018/2018-05-24-to-2018-10-30--cluster-meeting-archive.md [hunk 4]
```diff
@@ -0,0 +1,1250 @@
...
+- mSorted out license and moved gorpgorpc to libp2p/ plus update projects table plus move to codecov
+- Lots of work on the twitter pinbot
```
Comment (diasdavid) [inline meeting-notes/2018/2018-05-24-to-2018-10-30--cluster-meeting-archive.md]: Congratz on the release!! 👏🏽👏🏽👏🏽 The tweetersphere got super excited about it

File: meeting-notes/2018/2018-05-24-to-2018-10-30--cluster-meeting-archive.md (hunk 2 above)
Comment (lanzafame) [reply meeting-notes/2018/2018-05-24-to-2018-10-30--cluster-meeting-archive.md]: Nowhere as it didn't happen this week.

File: meeting-notes/2018/2018-05-24-to-2018-10-30--cluster-meeting-archive.md (hunk 3 above)
Comment (lanzafame) [reply meeting-notes/2018/2018-05-24-to-2018-10-30--cluster-meeting-archive.md]: https://github.com/ipfs/ipfs-cluster/pull/581

**Pull request merged**